File: Java/Human.java
public class Human {
    // Nama Human
    private String nama;
    // nik Human
    private String nik;
    // jenis kelamin Human
    private String Jenis_kelamin;


    Human(){} // konstruktor Human

    Human(String nama,String nik,String Jenis_kelamin) {
        // konstruktor langsung isi attribut
        this.nama = nama;
        this.nik = nik;
        this.Jenis_kelamin = Jenis_kelamin;
    }

    // set nama
    public void setNama(String namaHuman) {
        this.nama = namaHuman;
    }

    // get nama
    public String getNama() {
        return nama;
    }

    // set nik
    public void setNik(String nikHuman) {
        this.nik = nikHuman;
    }

    // get nik
    public String getNik() {
        return nik;
    }

    // set jenis kelamin
    public void setJenis_kelamin(String Jenis_kelamin) {
        this.Jenis_kelamin = Jenis_kelamin;
    }

    // get jenis kelamin
    public String getJenis_kelamin() {
        return this.Jenis_kelamin;
    }

}

File: C++/Human.cpp
// Import library.
#include <bits/stdc++.h>
// Using standard namespace.
using namespace std;

class Human
{
    // Membuat atribut privte
private:
    // nama Human
    string nama;
    // nik Human
    string nik;
    // jenis kelamin Human
    string Jenis_kelamin;

public:
    Human() {} // konstruktor Human

    Human(string nik, string nama, string Jenis_kelamin)
    {
        // konstruktor langsung isi attribut
        this->nama = nama;
        this->nik = nik;
        this->Jenis_kelamin = Jenis_kelamin;
    }

    // Setter and Getter methods for private attributes
    // set nama
    void setNama(string namaHuman)
    {
        this->nama = namaHuman;
    }

    // get nama
    string getNama()
    {
        return nama;
    }

    // set nik
    void setNik(string nikHuman)
    {
        this->nik = nikHuman;
    }

    // get nik
    string getNik()
    {
        return nik;
    }

    // set jenis kelamin
    void setJenis_kelamin(string Jenis_kelamin)
    {
        this->Jenis_kelamin = Jenis_kelamin;
    }

    // get jenis kelamin
    string getJenis_kelamin()
    {
        return this->Jenis_kelamin;
    }

    // Destruktor
    ~Human()
    {
    }
};